src/main/java/net/sf/jabref/model/entry/BibEntry.java

```diff
@@ -194,7 +202,6 @@ public void setType(EntryType type) {
     /**
      * Returns the contents of the given field, or null if it is not set.
      */
-    @Deprecated //Use getFieldOptional instead
```

Comment:
I don't see any reason why getField is necessary there. You can simply replace the null check in escapeValue https://github.com/JabRef/jabref/pull/1451/files#diff-044ee1a556d146810fd96d93f040505aR516 with optional.isPresent.